spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -288,7 +288,7 @@ private SpoonDeviceRunner getTestRunner(String serial, SpoonInstrumentationInfo
     private String methodName;
     private boolean noAnimations;
     private IRemoteAndroidTestRunner.TestSize testSize;
-    private int adbTimeout = DEFAULT_ADB_TIMEOUT;
+    private int adbTimeout = DEFAULT_ADB_TIMEOUT_SEC * 1000;
```

Comment:
Only issue with that is then either the public API needs to change (setAdbTimeoutMillis()) or the setter API won't match the field name.  I suppose we could add setAdbTimeoutMillis() and deprecate setAdbTimeout()...  or just be ok with the method/field naming difference.  Thoughts?